build/charts/antrea/crds/nodelatencymonitor.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: nodelatencymonitors.crd.antrea.io
+spec:
+  group: crd.antrea.io
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          required:
+            - spec
+          properties:
+            spec:
+              type: object
+              required:
+                - pingIntervalSeconds
+              properties:
+                pingIntervalSeconds:
+                  type: integer
+                  format: int32
+                  minimum: 1
+                  description: "Ping interval in seconds, must be at least 1."
+                  default: 10
```

Comment:
I thought we wanted a default value of `60`, that's what I see in the comments in `pkg/apis/crd/v1alpha1/types.go`. It's also better than `10` IMO.

Reply:
Ok, I will change it.